WordPress/src/main/java/org/wordpress/android/ui/posts/PageAndPostListPreviewStateHelper.kt

```diff
@@ -0,0 +1,83 @@
+package org.wordpress.android.ui.posts
+
+import androidx.fragment.app.FragmentActivity
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.ui.posts.RemotePreviewLogicHelper.RemotePreviewType
+import org.wordpress.android.ui.uploads.UploadService
+import org.wordpress.android.util.AppLog
+
```

Comment:
Tbh I'm not a big fan of functions. I think they are really useful for some scenarios, but I prefer to use methods wherever possible. I'd consider wrapping these functions with an injectable (@Reusable) class.

- Afaik functions can't be mocked which is a huge downside with very little/no gain. 
- The autocompletion can easily become very messy - making sure we don't have that many collisions in class names is already difficult enough, making sure we don't have collisions in function names is almost impossible.

Wdyt?

Reply:
Fixed in 59a4a3a2ff66e2514e627f09322a7602af44ed73